docs/instrumentation.md

```diff
@@ -0,0 +1,283 @@
+## TPM driver instrumentation
+
+For development purposes, a set of scripts was created that allows for the
```

Comment:
@acwiek Can we add `instrumentation` changes in a separate MR? It is more related to the SPI work (7th phase). While the rest of the changes are related to running with USB-CDC and verification of basic communication (1tst phase).

Reply:
created dedicated PR.
https://github.com/lpn-plant/lpntpm-docs/pull/7